homeassistant/components/airly/sensor.py

```diff
@@ -120,8 +121,10 @@ class AirlySensorEntityDescription(SensorEntityDescription):
         key=ATTR_API_CO,
         device_class=SensorDeviceClass.CO,
         name=ATTR_API_CO,
-        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
         state_class=SensorStateClass.MEASUREMENT,
+        # Convert API-reported µg/m³ to ppm
+        value=lambda value: round(value / 1.15 / 1000, 3),
```

Comment:
We should avoid conversions like these. Instead, as suggested in the linked original PR/comment: Consider opening an architectural issue to suggest support for adding other units?